appsec/src/extension/attributes.h

```diff
@@ -6,6 +6,8 @@
 #ifndef DD_ATTRIBUTES_H
 #define DD_ATTRIBUTES_H
 
+#include "configuration.h"
```

Comment:
Is this required?

Reply:
When attributes in included in a file without configuration.h included first it fails in the `build_packages` CI for some specific version. So it may be better to include it direcly in it to avoid issues, wdyt ?

Reply:
if without it fails, then it's required. Not sure if @Anilm3 has any suggestion

Reply:
I'm not sure about the issue described by @Leiyks, but the problem locally seems to be due to the introduction of `zai_str` into the `user_tracking.(c|h)` file, while simultaneously having removed `#include "configuration.h"`. Just add that header back to `user_tracking.h` and remove it from here.